packages/app/src/components/collapsible/collapsible-section.tsx

```diff
@@ -105,7 +105,7 @@ export const CollapsibleSection = ({
   id,
   hideBorder,
 }: CollapsibleSectionProps) => {
-  const [open, setOpen] = useState(true);
+  const [open, setOpen] = useState(false);
```

Comment:
@VWSCoronaDashboard10 I see you once explicitly changed this to `true`. Do I break any invisible functionality by changing it back? (it seems to work fine)

Reply:
While you're changing this code anyway maybe use a better name for the boolean. We normally prefix them so it's clear that it's a boolean. "open" can be anything, and I would expect it to be a function most of all. 

```suggestion
  const [isOpen, setIsOpen] = useState(false);
```

Reply:
If I recall correctly that is due to the fact when Javascript is disabled, they are by default open.

Reply:
yep that's the use case for initially open collapsibles

Reply:
Would be great to have that in a comment then :)